components/match2/wikis/rainbowsix/match_group_input_custom.lua

```diff
@@ -60,8 +60,7 @@ function CustomMatchGroupInput.processMatch(match, options)
 	match.finished = MatchGroupInputUtil.matchIsFinished(match, opponents)
 
 	if match.finished then
-		match.resulttype = MatchGroupInputUtil.getResultType(winnerInput, finishedInput, opponents)
-		match.walkover = MatchGroupInputUtil.getWalkover(match.resulttype, opponents)
+		match.status = MatchGroupInputUtil.getMatchStatus(winnerInput, finishedInput)
 		match.winner = MatchGroupInputUtil.getWinner(match.resulttype, winnerInput, opponents)
```

Comment:
resulttype should be unset by now, so why pass it along?
do we need to pass along status instead?